src/foam/nanos/crunch/README.md

```diff
@@ -0,0 +1,83 @@
+# CRUNCH User Guide and Documentation
+
+&nbsp;
+
+### CRUNCH - Continuous Reactive User Nano-Capability Hierarchy
+
+CRUNCH is a [Knowledge-Based Configuration System](https://en.wikipedia.org/wiki/Knowledge-based_configuration) that configures a user's account with various capabilities based upon the features and functionalities.
```

Comment:
Grammar for this sentence is odd at the end. The "based upon the features and functionalities" part.